src/main/java/org/ga4gh/starterkit/wes/controller/Runs.java

```diff
@@ -73,6 +77,7 @@ public RunsListResponse getRuns() throws JsonProcessingException {
             }
         }
         RunsListResponse runsListResponse = new RunsListResponse(runStatusArrayList);
+        loggingUtil.debug(listRunsQuery);
```

Comment:
I would add complete sentences for logs (This helps identify the location of the log in your code) and avoid adding the outputs especially if they might be long.
For example, instead of `loggingUtil.debug(listRunsQuery);` I would do `loggingUtil.debug("Returning response for getRuns endpoint");`

Reply:
Also, we could add `loggingUtil.debug("querying for list of runs. SQL query: " + listRunsQuery);` before line 63.